assets/js/components/surveys/SurveyQuestionRating.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * SurveyQuestionRating component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import SurveyHeader from './SurveyHeader';
+import SurveyQuestionRatingChoice from './SurveyQuestionRatingChoice';
+import IconSurveyUnhappy from '../../../svg/survey-unhappy.svg';
+import IconSurveyDissatisfied from '../../../svg/survey-dissatisfied.svg';
+import IconSurveyNeutral from '../../../svg/survey-neutral.svg';
+import IconSurveySatisfied from '../../../svg/survey-satisfied.svg';
+import IconSurveyDelighted from '../../../svg/survey-delighted.svg';
+
+const SurveyQuestionRating = ( { question, choices, answerQuestion, dismissSurvey } ) => (
+	<div className="googlesitekit-survey__question-rating">
+		<SurveyHeader
+			title={ question }
+			dismissSurvey={ dismissSurvey }
+		/>
+
+		<div className="googlesitekit-survey__body">
+			<div className="googlesitekit-survey__choices">
+				{ choices.map( ( choice, index ) => (
+					<SurveyQuestionRatingChoice
+						key={ index }
+						choice={ choice }
+						answerQuestion={ answerQuestion }
+						Icon={ SurveyQuestionRating.ordinalIconMap[ choice.answer_ordinal ] }
+					/>
+				) ) }
+			</div>
+		</div>
+	</div>
+);
+
+SurveyQuestionRating.ordinalIconMap = {
+	1: IconSurveyUnhappy,
+	2: IconSurveyDissatisfied,
+	3: IconSurveyNeutral,
+	4: IconSurveySatisfied,
+	5: IconSurveyDelighted,
+};
```

Comment:
Not sure we need to attach this to the component, but doesn't really hurt I suppose.

Alternatively, we could pass `ordinalIconMap` as a prop, and use this object as the default value. Just an idea.

Reply:
Actually good catch. Doesn't make sense to have the map in this component. I moved it over to the `SurveyQuestionRatingChoice` component. 👍